cmd/lncli/commands.go

```diff
@@ -1985,7 +1989,18 @@ func describeGraph(ctx *cli.Context) error {
 	client, cleanUp := getClient(ctx)
 	defer cleanUp()
 
-	req := &lnrpc.ChannelGraphRequest{}
+	var (
+		private bool
+	)
+
+	switch {
+	case ctx.IsSet("private"):
+		private = true
+	}
+
+	req := &lnrpc.ChannelGraphRequest{
+		Private: bool(private),
```

Comment:
As it's a boolean value we're attempting to retrieve, we can simply do `Private: ctx.Bool("private"),` and remove the code above.